cluster/node-pools/master-default/userdata.yaml

```diff
@@ -128,6 +128,11 @@ write_files:
           - --service-account-signing-key-file=/etc/kubernetes/ssl/service-account-private-key.pem
           - --service-account-issuer=kubernetes/serviceaccount
           {{- end }}
+          {{- if eq .Cluster.ConfigItems.teapot_admission_controller_service_account_iam "true" }}
```

Comment:
I was not sure about this so I created a new key. I think this is not required. Only one `service-account-signing-key-file` and corresponding `service-account-key-file` should be specified.

Reply:
You're right. Dropped that as well.